cves/CVE-2014-0117.yml

```diff
@@ -95,9 +105,14 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer:
-  code:
-  fix:
+  answer: |
+    Automated unit tests were most likely used on the code where the vulnerability
+    lies but not all the unit tests are not publicized on their repos, for good reason.
+    The fix, and even looking at the squashed commit that included the fix commit, there 
+    is no change to the available unit test that we can see from the repos.
+  code: 
```

Comment:
I think the code section is meant to have a "true/false" answer here, based on whether or not the original VCC code had unit tests associated with it. If you could not find any directly referencing the VCC code, I'd put it as false.